src/test/config.sol

```diff
@@ -95,15 +93,16 @@ contract Config {
     function setValues(address chief) public {
         // Add spells if there is a need to test prior to their cast() functions
         // being called on-chain. They will be executed in order from index 0.
-        address[] memory prevSpells = new address[](0);
+        address[] memory prevSpells = new address[](1);
+        prevSpells[0] = address(0x07D8D43916bc235B71d9683111De7c7c626Bb906);
```

Comment:
This has been cast already and can be removed if you like.

Reply:
This is being checked in the tests as well: it doesn't try to re-cast spells that already been cast, but I also removed it